src/cloud-api-adaptor/pkg/userdata/provision.go

```diff
@@ -12,11 +15,27 @@ import (
 	"github.com/confidential-containers/cloud-api-adaptor/src/cloud-providers/aws"
 	"github.com/confidential-containers/cloud-api-adaptor/src/cloud-providers/azure"
 	"github.com/confidential-containers/cloud-api-adaptor/src/cloud-providers/docker"
+	toml "github.com/pelletier/go-toml/v2"
 	"gopkg.in/yaml.v2"
 )
 
+const (
+	ConfigParent = "/run/peerpod"
+
+	AaCfgPath     = "/run/peerpod/aa.toml"
+	AgentCfgPath  = "/run/peerpod/agent-config.toml"
+	AuthJsonPath  = "/run/peerpod/auth.json"
+	CdhCfgPath    = "/run/peerpod/cdh.toml"
+	DaemonCfgPath = "/run/peerpod/daemon.json"
+
+	InitdataMeta = "/run/peerpod/initdata.meta"
+	CheckSumPath = "/run/peerpod/checksum.txt"
```

Comment:
what is the purpose of checksum path and initdata.meta? is this documented somewhere (how) would it be different from what's documented in the trustee init-data spec?

Reply:
Thanks @mkulke , Spec here https://github.com/confidential-containers/trustee/blob/main/kbs/docs/initdata.md mentioned initdata digest calculation but not cover this indeed, I'll open a PR to enhance the spec.

Reply:
https://github.com/confidential-containers/trustee/pull/450